components/data-table/private/header-cell.jsx

```diff
@@ -122,26 +122,36 @@ const DataTableHeaderCell = createReactClass({
 		const sortDirection = this.props.sortDirection || this.state.sortDirection;
 		const expandedSortDirection =
 			sortDirection === 'desc' ? 'descending' : 'ascending';
-		const ariaSort = isSorted ? expandedSortDirection : null;
+		const ariaSort = isSorted ? expandedSortDirection : 'none';
+		// The following padding styles exist in case you have
+		const thStyle = {
+			paddingLeft: sortable ? '0' : null,
```

Comment:
Maybe I'm missing something, but the `<th>` already has `.slds-table_resizable-col`, which includes 'padding: 0`?